homeassistant/components/image_processing/facebox.py

```diff
@@ -74,18 +97,18 @@ def process_image(self, image):
         try:
             response = requests.post(
                 self._url,
-                json=encode_image(image),
+                json={"base64": encode_image(image)},
                 timeout=9
                 ).json()
         except requests.exceptions.ConnectionError:
             _LOGGER.error("ConnectionError: Is %s running?", CLASSIFIER)
             response['success'] = False
 
         if response['success']:
-            faces = response['faces']
-            total = response['facesCount']
-            self.process_faces(faces, total)
-            self._matched = get_matched_faces(faces)
+            self.total_faces = response['facesCount']
```

Comment:
Why set the instance attribute here when that will be done again below by `process_faces`? Just save total faces as a local variable.

Reply:
ok